.github/actions/hugo-build-versioned-action/action.yaml

```diff
@@ -0,0 +1,39 @@
+name: Hugo Versioned Build
+description: Builds documentation using Hugo
+inputs:
+  base-url:
+    required: false
+    description: Optional base url used during build
+  working-directory:
+    required: false
+    description: Path to documentation directory
+    default: versioned_docs
+  hugo-env:
+    required: false
+    description: Environment variable
+    default: public
+
+runs:
+  using: "composite"
+  steps:
+    - name: Generate Versioned Documentation
+      shell: bash 
+      run: |
+        cd docs
+        ../scripts/generate.sh origin master keep_master
+    - name: Setup Hugo
+      uses: peaceiris/actions-hugo@v2
+      with:
+        hugo-version: '0.81.0'
+        extended: true
+    - name: Install dependencies
+      working-directory: docs
+      run: npm install
+      shell: bash
+    - name: Build
+      working-directory: docs
+      env:
+        HUGO_ENV: ${{ inputs.hugo-env }}
+        BASE_URL: ${{ inputs.base-url }}
+      run: hugo --minify ${BASE_URL:+--baseURL $BASE_URL}
+      shell: bash
```

Comment:
If I am not mistaken, this is only a repetition of our action. Why not to use it again? 

https://github.com/gooddata/gooddata-python-sdk/blob/master/.github/actions/hugo-build-action/action.yaml